index.js

```diff
@@ -76,13 +76,17 @@ var InfluxDB = function (options) {
 }
 
 InfluxDB.prototype._parseResults = function (response, callback) {
+  var self = this
   var results = []
   _.each(response, function (result) {
     var tmp = []
     if (result.series) {
       _.each(result.series, function (series) {
         var rows = _.map(series.values, function (values) {
-          return _.extend(_.zipObject(series.columns, values), series.tags)
+          var columnsDesanitized = _.map(series.columns, self._desanitizeString)
+          var valuesDesanitized = _.map(values, self._desanitizeString)
+          var tagsDesanitized = _.forOwn(series.tags, self._desanitizeObjectIter.bind(self))
```

Comment:
I'm not sure this is needed. In 0.13.0 it doesn't look like results of queries include the original escaping characters:

```js
> insert test_series,tag1=value\ b,tag\ 2=x value=2
> select * from test_series
{
    "results": [
        {
            "series": [
                {
                    "name": "test_series",
                    "columns": [
                        "time",
                        "tag 2",
                        "tag1",
                        "value"
                    ],
                    "values": [
                        [
                            1474519055673092238,
                            "x",
                            "value b",
                            2
                        ]
                    ]
                }
            ]
        }
    ]
}
```